packages/yarn-plugin-allow-scripts/README.md

```diff
@@ -1,15 +1,29 @@
 # @lavamoat/yarn-plugin-allow-scripts
 
-A Yarn v3 plugin for running `@lavamoat/allow-scripts` after install.
+A Yarn Berry plugin for integrating `@lavamoat/allow-scripts` lifecycle-script protection.
 
 ### Install
 
-Yarn v3 plugins are installed via public URL.
+#### Yarn v4
+
+First, perform basic [configuration](https://github.com/LavaMoat/LavaMoat/blob/main/packages/allow-scripts/README.md#configure) `@lavamoat/allow-scripts` for the package.
```

Comment:
this sentence seems to be missing something

Reply:
Good catch! Was missing an ` of`.